cache.go

```diff
@@ -0,0 +1,98 @@
+package wazero
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	goruntime "runtime"
+
+	"github.com/tetratelabs/wazero/api"
+	"github.com/tetratelabs/wazero/internal/compilationcache"
+	"github.com/tetratelabs/wazero/internal/version"
+	"github.com/tetratelabs/wazero/internal/wasm"
+)
+
+// CompileCache is the configuration for compilation cache behavior of wazero.Runtime and can be passed to wazero.RuntimeConfig.
```

Comment:
```suggestion
// CompileCache reduces time spent compiling (Runtime.CompileModule) the same wasm module.
//
// Instances of this can be reused across multiple runtimes, if configured via RuntimeConfig.
```